rust/ql/test/library-tests/dataflow/taint/TaintFlowStep.expected

```diff
@@ -1,5 +1,5 @@
-| file://:0:0:0:0 | [summary param] 0 in lang:alloc::_::crate::fmt::format | file://:0:0:0:0 | [summary] to write: ReturnValue in lang:alloc::_::crate::fmt::format | MaD:9 |
-| file://:0:0:0:0 | [summary param] self in lang:alloc::_::<crate::string::String>::as_str | file://:0:0:0:0 | [summary] to write: ReturnValue in lang:alloc::_::<crate::string::String>::as_str | MaD:7 |
+| file://:0:0:0:0 | [summary param] 0 in lang:alloc::_::crate::fmt::format | file://:0:0:0:0 | [summary] to write: ReturnValue in lang:alloc::_::crate::fmt::format | MaD:14 |
+| file://:0:0:0:0 | [summary param] self in lang:alloc::_::<crate::string::String>::as_str | file://:0:0:0:0 | [summary] to write: ReturnValue in lang:alloc::_::<crate::string::String>::as_str | MaD:12 |
```

Comment:
@hvitved this test is unrelated to the PR, apart from that they both use models-as-data.  I think that means we shouldn't be outputting these IDs in the tests?

Reply:
How is that done?

Reply:
We would have to pull out the relevant bits from [the `ShowProvenance` module](https://github.com/github/codeql/blob/fcf6c3c4e83f127aea9ce5aed3e0ceb4feb65bc5/shared/dataflow/codeql/dataflow/test/ProvenancePathGraph.qll#L91).

Reply:
Do you want to do this, or shall I do this as a follow-up PR?

Reply:
Let's do it on a separate PR.